### PR TITLE
Cleanup tests

### PR DESCRIPTION
**Pull request by grouma**
- Move screenshot test over to package:dwds where it belongs
- No longer manually create a debug service but make use of public package:dwds APIs

Towards https://github.com/dart-lang/webdev/issues/475

### CODE REVIEW

File: dwds/lib/src/dwds_vm_client.dart [hunk 1]
```diff
@@ -65,15 +65,15 @@ class DwdsVmClient {
       await chromeProxyService.tabConnection.sendCommand('Page.reload');
       return {'result': Success().toJson()};
     });
-    await client.registerService('fullReload', 'WebDev');
+    await client.registerService('fullReload', 'DWDS');
 
-    client.registerServiceCallback('ext.webdev.screenshot', (_) async {
+    client.registerServiceCallback('ext.dwds.screenshot', (_) async {
```
Comment (jakemac53) [inline dwds/lib/src/dwds_vm_client.dart]: cc @jonahwilliams not sure if you were using this yet

File: webdev/test/serve/screenshot_test.dart [hunk 2]
```diff
@@ -1,60 +0,0 @@
-// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
```
Comment (jakemac53) [inline webdev/test/serve/screenshot_test.dart]: Why was this deleted?

Comment (grouma) [reply webdev/test/serve/screenshot_test.dart]: It was moved over to package:dwds. No need to test from webdev.

Comment (jakemac53) [reply webdev/test/serve/screenshot_test.dart]: I don't see a test in dwds though (would have expected this to be a move?)

Comment (grouma) [reply webdev/test/serve/screenshot_test.dart]: Ahh good catch. Didn't include it in my commit.

File: dwds/lib/src/dwds_vm_client.dart (hunk 1 above)
Comment (jonahwilliams) [reply dwds/lib/src/dwds_vm_client.dart]: Haven't landed anything yet, so no issues here
